manager-bundle/src/ContaoManager/Plugin.php

```diff
@@ -403,7 +427,8 @@ private function getDatabaseUrl(ContainerBuilder $container): string
         }
 
         return sprintf(
-            'mysql://%s%s:%s%s',
+            '%s://%s%s:%s%s',
+            $useMysqli ? 'mysqli' : 'mysql',
```

Comment:
Are these really the only two variants? What is the driver name for PostgreSQL for example?

Reply:
we don't support anything except mysql as far as I know (we didn't previously either).

Reply:
Not right now, however, we might in the future. Therefore if there is an easy way to extract the driver from the URL, we should use it.

Reply:
🤷

Reply:
IMHO we should just use `$extensionConfig['dbal']['connections']['default']['driver']` as driver. This should work with Doctrine: https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#driver

Reply:
```suggestion
            array_reverse(array_filter(array_map(function ($extensionConfig) {
                return $extensionConfig['dbal']['connections']['default']['driver'] ?? null;
            }, $extensionConfigs))[0] ?? 'mysql',
```